test/mongodb.ts

```diff
@@ -143,6 +143,7 @@ export * from '../src/cursor/list_indexes_cursor';
 export * from '../src/cursor/run_command_cursor';
 export * from '../src/db';
 export * from '../src/deps';
+export * as deps from '../src/deps';
```

Comment:
This isn't really the correct way for us to maintain this file, the list is generated from the function at the top of the file and it's intended to make all internally exported functions available at the top level. (so mongodb.saslprep is exported already). So this is supposed to mirror the driver,  adding this adds a `deps` named export to our tests that doesn't exist for the driver

Reply:
```suggestion
```

Now that we have it in the test file